### PR TITLE
{bio}[GCC/8.3.0] DIAMOND v0.9.30

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/761a4a839db8f8139e90a59068d3a87e for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/343a617939b611d5ff723bbe969d597e for a full test report.

File: easybuild/easyconfigs/d/DIAMOND/DIAMOND-0.9.30-GCC-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+easyblock = 'CMakeMake'
```
Comment (verdurin) [inline easybuild/easyconfigs/d/DIAMOND/DIAMOND-0.9.30-GCC-8.3.0.eb]: @lexming Please reinstate the original attribution, then add a comment to note your own changes.

Comment (lexming) [reply easybuild/easyconfigs/d/DIAMOND/DIAMOND-0.9.30-GCC-8.3.0.eb]: I updated `DIAMOND-0.9.24-iccifort-2019.1.144-GCC-8.2.0-2.31.1.eb` which does not have any attribution to it.

Comment (verdurin) [reply easybuild/easyconfigs/d/DIAMOND/DIAMOND-0.9.30-GCC-8.3.0.eb]: @lexming sure, but if you look at the `foss` versions it was still there. Not a vital thing, but I think it's a shame to remove these attributions.

Comment (lexming) [reply easybuild/easyconfigs/d/DIAMOND/DIAMOND-0.9.30-GCC-8.3.0.eb]: I see, I don't mind adding back the original attribution but I won't add any other comment attributing changes to authors as I cannot track all the updates to the various easyconfigs of `DIAMOND` in the past years.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node377.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/5738b2235363a7aab91a723f9ffc3493 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @lexming!

**Pull request merged**